cluster/ci/config/prow/plugins.yaml

```diff
@@ -610,6 +613,8 @@ config_updater:
       name: labels
     cluster/ci/config/secret-mirroring/mapping.yaml:
       name: secret-mirroring
+    ci-operator/jobs/openshift/external-dns/*openshift*.yaml:
```

Comment:
These are no bueno and will match other branches I think. Why do you need an `openshift` branch? Why not use `openshift-4.0` etc?

/hold

Reply:
Yup, can definitely change to `openshift-4.0`. The intent here is to have a branch that includes our CI stuff and keep `master` as an exact replica of the original repo.